tuning-profiles/custom-hiera-ex-large-128G.yaml

```diff
@@ -0,0 +1,60 @@
+# Tuning suitable for a Satellite with 128GG of RAM
```

Comment:
typo here: s/GG/G/

Reply:
I'm really in favor of basing the tuning on memory sizes. Customers should pick their hardware based on their needs and client load, then pick the tuning that matches the *hardware*. 

I think this simplifies picking which tuning to apply as some customers may need 64G for 10,000 hosts because their usage profile is very heavy whereas you may see a 40,000 host Satellite that runs fine on 64G because they only use it for a minimal set of functionality.

Reply:
fixed typo as well..